Various grammer and typo nits (#85)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

Producing a Transparent Statement may be considered a form of notarization.
A Statements payload content MAY be encrypted and opaque to the Transparency Services, if so desired: however the metadata MUST be transparent in order to warrant trust for later processing.
Transparent Statements provide a common basis for holding Issuers accountable for the Statement payload about Artifacts they release and (more generally) principals accountable for auxiliary Signed Statements from other Issuers about the original Signed Statement about an Artifact.
Issuers may Register new Signed Statements about Artifacts, but they cannot delete or alter Signed Statements previously added to the append-only Log.
A Transparency Service may restrict access to Signed Statements through access control policies. However, third parties (such as Auditors) would be granted access as needed to attest to the validity of the Artifact, Feed or the entirety of the Transparency Service.

> Trust in the Transparency Service itself is supported both by protecting their implementation (using, for instance, replication, trusted hardware, and remote attestation of system state) and by enabling independent audits of the correctness and consistency of its Registry, thereby holding the organization that operates it accountable.

Reviewer: ```suggestion
Trust in the Transparency Service itself is supported both by protecting their implementation (using, for instance, replication, trusted hardware, and remote attestation of a system's operational state) and by enabling independent audits of the correctness and consistency of its Registry, thereby holding the organization that operates it accountable.
```

Reviewer: nitpicking

Unlike CT, where independent Auditors are responsible for enforcing the consistency of multiple independent instances of the same global Registry, each Transparency Service is required to guarantee the consistency of its own Registry (for instance, through the use of a consensus algorithm between replicas of the Registry), but assume no consistency between different Transparency Services.

> Breadth of access is critical so the Transparency Services specified in this architecture cater to two types of audiences:

Reviewer 2: ```suggestion
Breadth of access is critical so the Transparency Service specified in this architecture cater to two types of audiences:
```

Issuer:

: an entity that creates Signed Statements about software Artifacts in the supply chain.

> An Issuer may be the owner or author of Artifacts, or an independent third party such as a reviewer or an endorser.

Reviewer 2: @JAG-UK I think software was deliberately added as the current scope was only `software`

Reviewer 2: Sorry I just realized that software is already used in 194 so fine with it: now.

The most important is maintaining a Registry, the verifiable data structure that records Signed Statements, and enforcing a Registration Policy.
It also maintains a service key, which is used to endorse the state of the Registry in Receipts.
All Transparency Services MUST expose standard endpoints for Registration of Signed Statements and Receipt issuance, which is described in {{sec-messages}}.

> Each Transparency Service also defines its Registration Policies, which MUST apply to all entries in the Registry.

Reviewer 2: ```suggestion
Each Transparency Service also defines its own Registration Policies, which MUST apply to all entries in the Registry.
```

Note that the SCITT Architecture does not require trust in a single centralized Transparency Service: different actors may rely on different Transparency Services, each registering a subset of Signed Statements subject to their own policy.

In both cases, the SCITT Architecture provides generic, universally-verifiable cryptographic proof to individually blame Issuers or the Transparency Service.

> On the one hand, this enables valid actors to detect and disambiguate malicious actors who issue contradictory Signed Statements to different entities (Verifiers, Auditors, Issuers), otherwise known as 'equivocation'.

Reviewer 2: ```suggestion
On one hand, this enables valid actors to detect and disambiguate malicious actors who issue contradictory Signed Statements to different entities (Verifiers, Auditors, Issuers), otherwise known as 'equivocation'.
```
